main.go

```diff
@@ -13,7 +13,7 @@
 
 package main
 
-import "github.com/schrej/godacov/cmd"
+import "github.com/stlimtat/godacov/cmd"
```

Comment:
Could you revert this so it doesn't fetch the package from two different repositories please?